variable.go

```diff
@@ -433,11 +434,33 @@ func (vr *variableResolver) resolve(ctx *ExecutionContext) (*Value, error) {
 				currArgs = append([]functionCallArgument{executionCtxEval{}}, currArgs...)
 			}
 
+			// If kwargs map is needed
+			includeKwargsBit := 0
+			if t.NumIn() > 0 && t.In(0) == typeOfKwargsMap {
+				includeKwargsBit = 1
+			}
+
+			evaluatedArgs := make([]*Value, 0)
+			kwargCount := 0
+			for _, arg := range currArgs {
+				v, err := arg.Evaluate(ctx)
+				if err != nil {
+					return nil, err
+				}
+				if v.IsKwarg() {
+					if !(includeKwargsBit == 1) {
+						return nil, fmt.Errorf("calling a function using a keyword argument: %v=%v, but the function does not support kwargs", v.Name(), v)
```

Comment:
Should also tell the solution. "Add a map as first arg in the function (also give name of function for clarity)"